cmd/utils_test.go

```diff
@@ -78,11 +78,12 @@ func Test_copyGenesisFile_failure(t *testing.T) {
 
 func Test_createSidecar_success(t *testing.T) {
 	subnetName := "TEST_subnet"
+	tokenName := "TEST"
 	sidecarFile := subnetName + sidecar_suffix
 	const vm = models.SubnetEvm
 
 	// Write sidecar
-	err := createSidecar(subnetName, vm)
+	err := createSidecar(subnetName, vm, tokenName)
```

Comment:
We might want to add a small test case when `tokenName` is `""`

Reply:
Added that test, also changed the behavior slightly when tokenName is "". Might change it when I integrate tokenName's more fully in a follow-up PR.